docs/solutions/hypervisors/kvm/kvm-nested/terraform.tfvars

```diff
@@ -0,0 +1,48 @@
+###############################
+#### Environment variables ####
+###############################
+
+tenancy_ocid="ocid1.tenancy.oc1..abcabcababccbcbaabcabababcbabababcbababcbabababacbcabcabcabac4uq"
```

Comment:
Please switch these to placeholder text, similar to what is done in env-vars files for other examples such as https://github.com/oracle/terraform-provider-oci/blob/master/docs/solutions/rhel74_image/env-vars. 

Generally, if there's a sensible default that the user could realistically use, then set it to that. If the user must set a value of their own (as with tenancy_ocid, user, fingerprint, etc), then set placeholder text.

Reply:
@briangustafson I fixed all the code according to all your recommendations. I replaced the tfvar files by env-vars. Now It's look like OK. Thanks for your time reviewing the code.